tests/run_model_templates_tests.py

```diff
@@ -154,6 +156,24 @@ def _add_counter_to_rows(rows):
 
     return new_rows
 
+def save_list_to_path(all_tests, path):
```

Comment:
Could you remove `_to_path` and clarify which `list`

Reply:
Fixed